tests/rpc-tests/rpc-compat.js

```diff
@@ -0,0 +1,199 @@
+import fs from 'fs'
+import path from 'path'
+import { maxRetries, runScpCommand, runSshCommand, runSshCommandWithReturn } from '../../src/express/common/remote-worker'
+import assert from 'assert'
+import { checkAndReturnVMIndex, loadDevnetConfig } from '../../src/express/common/config-utils'
+import HDWalletProvider from '@truffle/hdwallet-provider'
+import { fundAccount } from '../test-eip-1559'
+import { timer } from '../../src/express/common/time-utils'
+const axios = require('axios/dist/node/axios.cjs')
+const Web3 = require('web3')
+
+require('dotenv').config({ path: `${process.cwd()}/.env` })
+
+async function initWeb3(machine) {
+  const provider = new HDWalletProvider({
+    mnemonic: {
+      phrase: process.env.MNEMONIC
+    },
+    providerOrUrl: `http://${machine}:8545`
+  })
+
+    return new Web3(provider)
+}
+
+async function init() {
+  console.log('📍Executing RPC compatibility tests')
+  const doc = await loadDevnetConfig('remote')
+  let machine = doc.devnetBorHosts[0]
+  let user = doc.devnetBorUsers[0]
+  let ip = `${user}@${machine}`
+  const borStartScriptLocation = '~/node/bor-start.sh'
+
+  const walletDisableFlagCmd = `grep -q 'disable-bor-wallet=false' ${borStartScriptLocation} && echo 'found' || echo 'not found'`
+  const isPresent = await runSshCommandWithReturn(ip, walletDisableFlagCmd, maxRetries)
+ 
+  if (isPresent === 'not found') {
```

Comment:
Are we sure type `Readable` (`isPresent`) can be compared with a `string` (`not found`). Otherwise the condition would always be `false`

Reply:
It is comparable because `runSshCommandWithReturn` returns a `string` (or `undefined`)

Reply:
It actually returns a Readable object, that is a string in your case. Not sure if I would introduce a `toString()`

Reply:
It doesn't seem necessary because the `if` block only gets executed once when you run it for the first time.